string group_name

uint8 SCORING_BOOL = 0
uint8 SCORING_UINT = 1
uint8[] types

string[] descriptions

int32[] current_values
